packages/html/src/SimpleTable.ts

```diff
@@ -25,16 +26,21 @@ export class SimpleTable extends HTMLWidget {
 
     update(domNode, element) {
         super.update(domNode, element);
+        const context = this;
         this._table
             .style("width", this.autoWidth() ? "auto" : "100%")
             ;
         const theadTrSelection = this._theadRow.selectAll("th").data(this.columns());
-        theadTrSelection.enter()
+        const theadTrUpdate = theadTrSelection.enter()
             .append("th")
             .attr("class", (n, i) => `th-${i}`)
             .merge(theadTrSelection)
-            .text(_d => (_d).toString())
             ;
+        if (this.renderHtmlDataCells()) {
+            theadTrUpdate.html(d => (d).toString());
+        } else {
+            theadTrUpdate.text(d => (d).toString());
```

Comment:
Why is `(d)` in the parens?

Reply:
I'm not sure. It was that way when I started working on it and I left it that way. It should be harmless to remove it.